lib/viberroo/bot.rb

```diff
@@ -140,8 +140,11 @@ def remove_webhook
     # @see https://viber.github.io/docs/tools/keyboards/#buttons-parameters
     # @see https://developers.viber.com/docs/api/rest-bot-api/#keyboards
     #
-    def send_message(message, keyboard: {})
-      request(URL::MESSAGE, { receiver: @response.user_id }.merge(message, keyboard))
+    def send_message(message:, keyboard: {})
```

Comment:
When called, `send_message({ ... })` reads better than `send_message(message: { ... })` - less redundant duplication.

Additionally, I would rather keep the interface without breaking changes if possible